classes/class-wc-connect-functions.php

```diff
@@ -95,5 +95,130 @@ public static function user_can_manage_labels() {
 			 */
 			return apply_filters( 'wcship_user_can_manage_labels', current_user_can( 'manage_woocommerce' ) || current_user_can( 'wcship_manage_labels' ) );
 		}
+
+		/**
+		 * Exports existing tax rates to a CSV and clears the table.
+		 *
+		 * Ported from TaxJar's plugin.
+		 * See: https://github.com/taxjar/taxjar-woocommerce-plugin/blob/42cd4cd0/taxjar-woocommerce.php#L75
+		 *
+		 * @return boolean
+		 */
+		public static function backup_existing_tax_rates() {
+			global $wpdb;
+
+			// Export Tax Rates
+			$rates = $wpdb->get_results(
+				$wpdb->prepare(
+					"SELECT * FROM {$wpdb->prefix}woocommerce_tax_rates
+			ORDER BY tax_rate_order
+			LIMIT %d, %d
```

Comment:
```suggestion
			        ORDER BY tax_rate_order
			        LIMIT %d, %d
```
I think this can be a bit tidier.